syntheseus/interface/molecule.py

```diff
@@ -81,3 +82,7 @@ def rdkit_mol(self) -> Chem.Mol:
         if "rdkit_mol" not in self.metadata:
             self.metadata["rdkit_mol"] = Chem.MolFromSmiles(self.smiles)
         return self.metadata["rdkit_mol"]
+
+
+def molecule_bag_to_smiles(mols: Bag[Molecule]) -> str:
+    return SMILES_SEPARATOR.join(mol.smiles for mol in mols)
```

Comment:
I noticed you are not sorting the molecules (while my previous string method sorted them). Is this because the contents of a `Bag` will always be sorted? If so, maybe clarify in a docstring that this should produce a consistently-ordered string.